build/debian-base/Makefile

```diff
@@ -16,24 +16,24 @@ all: all-build
 
 REGISTRY ?= staging-k8s.gcr.io
 IMAGE ?= $(REGISTRY)/debian-base
-BUILD_IMAGE ?= debian-build
 
 TAG ?= v2.0.0
 
-TAR_FILE ?= rootfs.tar
 ARCH?=amd64
+SKIP_REGISTER_MULTIARCH?=
 ALL_ARCH = amd64 arm arm64 ppc64le s390x
 
 TEMP_DIR:=$(shell mktemp -d)
 QEMUVERSION=v2.9.1
```

Comment:
Out of curiosity, why not bump the version of qemu in this PR ? Lot of improvements have been made in the last releases : https://www.qemu.org/2019/04/24/qemu-4-0-0/.

Reply:
I thought about that, but I would rather bump the version in a separate PR than "hiding" that upgrade in this "refactoring PR".